backend/qemu.pm

```diff
@@ -874,6 +874,13 @@ sub start_qemu {
         for (my $i = 0; $i < $num_networks; $i++) {
             if ($vars->{NICTYPE} eq "user") {
                 my $nictype_user_options = $vars->{NICTYPE_USER_OPTIONS} ? ',' . $vars->{NICTYPE_USER_OPTIONS} : '';
+                if ($vars->{QEMU_ENABLE_SMBD}) {
+                    my $smbd_bin = find_bin('/usr/sbin/', 'smbd');
+                    $smbd_bin                             or die "No server to provide SMB/CIFS services to SUT";
+                    (qx($smbd_bin --version) =~ /(\d.*)/) or die "smbd version was not matched";
```

Comment:
The error message leaves the question which version was expected. However, this check doesn't really validate whether the version matches some expected version (a digit followed by anything counts as version) which is also a bit weird.

Reply:
Ah, I just want to print the version of samba in case of issues, so we know what version has been used. On the second thought `die`ing is the best idea. :) Thanks for the point!